spec/factories/blood_pressures.rb

```diff
@@ -9,6 +9,26 @@
     association :facility, strategy: :build
     association :patient, strategy: :build
     user
+
+    trait :critical do
+      systolic { rand(180..240) }
```

Comment:
No need to make these random; just set them statically

Reply:
Makes sense.